Spring-2019/trees_graphs/SOLUTIONS_2019Spring.md

```diff
@@ -76,6 +76,40 @@ An example of a binary search tree that contains characters is:
 
 ### What is a graph?
 
+A graph is also a data structure that organizes and shows relationships 
+between individual data nodes. However, a graph is not organized in a 
+heirarchal way the way a tree is. Nodes in a graph are not required 
```

Comment:
Might be worth noting that a Tree is generally just an undirected graph.

Reply:
Addressed in 704a50576a2bded0c083637c42955e7fa328bd69